src/resources/temaCarte.java

```diff
@@ -0,0 +1,95 @@
+package resources;
+
+public class temaCarte
```

Comment:
vad ca ai mai multe clase in acelasi fisier, incearca sa le pui in fisiere separate